cloud/store/store.go

```diff
@@ -89,9 +91,7 @@ type Store struct {
 
 	nodeID   string
 	host     string
-	schema   *schema
-	db       DB
-	parser   Parser
+	db       localDB
```

Comment:
We call both `DB` the localDB and the indexer in different part of the package. Could we get a better name to properly differentiate them and avoid confusion when reading the code which one is doing which ?

Reply:
We only have localDB, DB doesn't exist anymore